File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/cereri-concediu-report-row.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class CereriConcediuReportRowModel {

	@JsonProperty("angajat", String)
    public angajat: string = null;

	@JsonProperty("tipConcediu", String)
    public tipConcediu: string = null;
    
	@JsonProperty("dataInitiere", JsonDateConverter)
	public dataInitiere: Date = null;
    
	@JsonProperty("dataInceput", JsonDateConverter)
	public dataInceput: Date = null;
    
	@JsonProperty("dataSfarsit", JsonDateConverter)
	public dataSfarsit: Date = null;

	@JsonProperty("status", String)
	public status: string = null;
    
	@JsonProperty("motivRespingere", String)
	public motivRespingere: string = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/CereriConcediuReportRowModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class CereriConcediuReportRowModel {

	private String angajat;
	private String tipConcediu;
	private Date dataInitiere;
	private Date dataInceput;
	private Date dataSfarsit;
	private String status;
	private String motivRespingere;

	public String getAngajat() {
		return angajat;
	}

	public void setAngajat(String angajat) {
		this.angajat = angajat;
	}

	public String getTipConcediu() {
		return tipConcediu;
	}

	public void setTipConcediu(String tipConcediu) {
		this.tipConcediu = tipConcediu;
	}

	public Date getDataInitiere() {
		return dataInitiere;
	}

	public void setDataInitiere(Date dataInitiere) {
		this.dataInitiere = dataInitiere;
	}

	public Date getDataInceput() {
		return dataInceput;
	}

	public void setDataInceput(Date dataInceput) {
		this.dataInceput = dataInceput;
	}

	public Date getDataSfarsit() {
		return dataSfarsit;
	}

	public void setDataSfarsit(Date dataSfarsit) {
		this.dataSfarsit = dataSfarsit;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getMotivRespingere() {
		return motivRespingere;
	}

	public void setMotivRespingere(String motivRespingere) {
		this.motivRespingere = motivRespingere;
	}

}
